### PR TITLE
#63 Github Markdown to HTML

### PR DESCRIPTION
**Pull request by mattjegan**
Implements #63 
* Added [marked](https://github.com/chjj/marked) as a dependancy to convert markdown to html
* Added feature where if a link (in markdown) goes over the line limit, we still include it since it will get shortened when converted to html.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -120,8 +121,19 @@ function getIssues(){
...
+                // Check if there is a link in the description
+                // if it starts before 120 chars, we include the
+                // whole link.
+                var link_match = /\[.+\]\(.+\)/;
+                var matched = description.match(link_match);
+                var match_index = description.search(link_match);
+                if (matched && match_index < 120) {
+                    description = description.substring(0, match_index) + matched + "...";
+                } else {
+                    description = description.substring(0, 120) + "...";
```
Comment (jenkoian) [inline index.js]: Extremely nit-picky but can we use `…` or `&hellip;` if it needs that

**Comment by jenkoian** [comment]
Awesome job dude - one minor thing over my ellipses ocd and should be good to go 👍

**Comment by mattjegan** [comment]
@jenkoian Thanks for letting me know about `&hellip;`

**Comment by jenkoian** [comment]
Cheers!

**Pull request merged**